### PR TITLE
Adds Onchain Summer Landing page for builders

### PR DESCRIPTION
**Pull request by zencephalon**
**What changed? Why?**

Adds new page.

**Notes to reviewers**

**How has it been tested?**

Extensively by manual testers.

### CODE REVIEW

File: apps/web/src/components/OnchainSummer/Brit.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default function Brit({ children, axis = 0 }: { children: React.ReactNode; axis?: number }) {
+  return (
+    <span
+      // className="font-britney text-[1.65rem]"
+      // scale britney to match coinbase-display
+      className="font-britney text-[86%]"
+      style={{ fontVariationSettings: `"styl" ${axis}` }}
```
Comment (ricardoMogg) [inline apps/web/src/components/OnchainSummer/Brit.tsx:7]: is this alright or is this mising an e?

File: apps/web/src/components/OnchainSummer/ResourcesBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+export default function ResourcesBlock() {
+  return (
+    <div className="my-12 flex w-full max-w-[1200px] flex-col px-8">
+      <div className="text-l flex flex-row items-center gap-2 font-mono uppercase">
+        <BlackCircle /> Resources
+      </div>
+      <FadeInSection>
+        <div className="my-6 flex flex-col gap-6">
+          <span className="text-5xl font-extrabold leading-9 md:text-7xl">
+            <Brit axis={147}>g</Brit>ET STARTE<Brit axis={147}>d</Brit>
```
Comment (ricardoMogg) [inline apps/web/src/components/OnchainSummer/ResourcesBlock.tsx:43]: possible typo

**Review by ricardoMogg — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for ricardoMogg @ 2024-05-15 20:23:18 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by wbnns — DISMISSED** [review]
utACK

**Review by qiwu7 — DISMISSED** [review]

**Review by wbnns — APPROVED** [review]

**Pull request merged**